internal/runners/activate/activate.go

```diff
@@ -2,6 +2,7 @@ package activate
 
 import (
 	"fmt"
+	"github.com/ActiveState/cli/internal/runbits/rtusage"
```

Comment:
Nit: tooling should organize this import

Reply:
There are a lot of out-of-place imports in the PR...

Reply:
Hmm I've been developing using Goland on Windows.. Seems I may have a setting or plugin on mac that I'm missing. Good eye!